yarn.lock

```diff
@@ -324,6 +324,54 @@
     "@smithy/util-waiter" "^3.1.2"
     tslib "^2.6.2"
 
+"@aws-sdk/client-ses@^3.637.0":
```

Comment:
Seems to be a big lock file change - is it all down to the additional package or might your yarn / node versions be out of sync